plugins/experimental/rate_limit/ip_reputation.cc

```diff
@@ -135,6 +135,8 @@ SieveLru::parseYaml(const YAML::Node &node)
   Dbg(dbg_ctl, "\twith perma-block rule: %s(%u, %u, %ld)", _name.c_str(), _permablock_limit, _permablock_threshold,
       static_cast<long>(_permablock_max_age.count()));
 
+  _initialized = true;
```

Comment:
@zwoop is this good place to set the flag? if any better place you imagined, let me know.

Reply:
Yeh, this feels right.